chirp/drivers/icr6.py

```diff
@@ -0,0 +1,794 @@
+            rset = RadioSetting("pgmscanedge/%d.lofreq" % kx,
+                                f"-- Scan {kx} Low Limit", rx)
+            # rset.set_apply_callback(myset_frqflgs, _pses, kx, "loflags",
+            #                        "lofreq")
+            edges.append(rset)
+
+            rx = RadioSettingValueFloat(0.1, 1309.995, fhigh, 0.010, 6)
+            rset = RadioSetting("pgmscanedge/%d.hifreq" % kx,
+                                f"-- Scan {kx} High Limit", rx)
+            # rset.set_apply_callback(myset_frqflgs, _pses, kx, "hiflags",
+            #                        "hifreq")
+            edges.append(rset)
+
+            # Tuning Step
+            # p_ts = ["-", 5, 6.25, 8.33, 9, 10, 12.5, 15, 20,
+            #        25, 30, 50, 100, 125, 200]
+
+            # Attenuation
+            # ndxt = 0
+            # ndxm = 0
+            # bxnd = 0
+            # tsopt = ["-", "5k", "6.25k", "10k", "12.5k", "15k",
+            #         "20k", "25k", "30k", "50k"]
+            # mdopt = ["-", "FM", "FM-N"]
+            # if fhigh > 0:
+            #    if fhigh < 135.0:  # Air band
+            #       bxnd = 1
+            #       tsopt = ["-", "8.33k", "25k", "Auto"]
+            #        ndxt = _pses[kx].tstp
+            #        if ndxt == 0xe:     # Auto
+            #            ndxt = 3
+            #        elif ndxt == 8:     # 25k
+            #            ndxt = 2
+            #        elif ndxt == 2:     # 8.33k
+            #            ndxt = 1
+            #        else:
+            #            ndxt = 0
+            #        mdopt = ["-"]
+            #    elif (flow >= 137.0) and (fhigh <= 174.0):   # VHF
+            #        ndxt = _pses[kx].tstp - 1
+            #        ndxm = _pses[kx].mode + 1
+            #        bxnd = 2
+            #    elif (flow >= 375.0) and (fhigh <= 550.0):  # UHF
+            #        ndxt = _pses[kx].tstp - 1
+            #        ndxm = _pses[kx].mode + 1
+            #        bxnd = 3
+            #    else:   # Mixed, ndx's = 0 default
+            #        tsopt = ["-"]
+            #        mdopt = ["-"]
+            #        bxnd = 4
+            #    if (ndxt > 9) or (ndxt < 0):
+            #        ndxt = 0   # trap ff
+            #    if ndxm > 2:
+            #        ndxm = 0
+            # # end if fhigh > 0
+            # rx = RadioSettingValueList(tsopt, tsopt[ndxt])
+            # rset = RadioSetting("pgmscanedge/%d.tstp" % kx,
+            #                    "-- Scan %d Freq Step" % kx, rx)
+            # rset.set_apply_callback(myset_tsopt, _pses, kx, "tstp", bxnd)
+            # edges.append(rset)
+
+            # rx = RadioSettingValueList(mdopt, mdopt[ndxm])
+            # rset = RadioSetting("pgmscanedge/%d.mode" % kx,
+            #                    "-- Scan %d Mode" % kx, rx)
+            # rset.set_apply_callback(myset_mdopt, _pses, kx, "mode", bxnd)
+            # edges.append(rset)
+        # End for kx (Edges)
```

Comment:
I'd really rather us not merge with all this commented-out code. I assume this is TODO work. Before we merge can you strip this out into a separate commit you can iterate on later?

Reply:
Done